frontend/js/stats.js

```diff
@@ -179,6 +193,146 @@ const fillRunTab = async (selector, data, parent = '') => {
+                case 'exception':
+                    typeTooltip = 'An error occurred during execution';
+                    break;
+                default:
+                    typeTooltip = 'Logs from an unknown or custom execution type';
+            }
+            const typeLabel = `<div class="ui purple label" data-tooltip="${typeTooltip}" data-position="top center"><i class="${typeIcon} icon"></i> ${typeTitle}</div>`;
+
+            const phaseLabel = logEntry.phase ?
+                `<div class="ui blue label" data-tooltip="Execution phase when this command was run" data-position="top center"><i class="clock icon"></i> ${escapeString(logEntry.phase)}</div>` : '';
+
+            const flowLabel = logEntry.flow ?
+                `<div class="ui green label" data-tooltip="Flow this command belongs to" data-position="top center"><i class="sitemap icon"></i> ${escapeString(logEntry.flow)}</div>` : '';
+
+            const idLabel = `<div class="ui label" data-tooltip="Unique identifier for this log entry" data-position="top center"><i class="hashtag icon"></i> ID: ${escapeString(logEntry.id)}</div>`;
+
+            const stdoutContent = logEntry.stdout ?
+                stdoutTemplate.replace('{{stdout}}', escapeString(logEntry.stdout)) : '';
+
+            const stderrContent = logEntry.stderr ?
+                stderrTemplate.replace('{{stderr}}', escapeString(logEntry.stderr)) : '';
+
+            // Add operation label for exceptions with specific tooltips
+            let operationLabel = '';
+            if (logEntry.type === 'exception') {
+                let operationTooltip;
+                switch (logEntry.cmd) {
+                    case 'run_scenario':
+                        operationTooltip = 'Exception occurred during main scenario execution runtime';
+                        break;
+                    case 'post_process':
+                        operationTooltip = 'Exception occurred during cleanup and post-processing phase';
+                        break;
+                    default:
+                        operationTooltip = `Exception occurred during ${logEntry.cmd} operation`;
+                }
+                const operationTitle = escapeString(logEntry.cmd.replace('_', ' ').replace(/\b\w/g, l => l.toUpperCase()));
+                operationLabel = `<div class="ui red label" data-tooltip="${operationTooltip}" data-position="top center"><i class="cogs icon"></i> ${operationTitle}</div>`;
+            }
+
+            // For exceptions, don't show the command section since it's now a label
+            const commandContent = logEntry.type === 'exception' ? '' :
+                `<div class="content">
+                    <h5 class="ui header"><i class="terminal icon"></i> Command</h5>
+                    <div class="ui segment">
+                        <code>${escapeString(logEntry.cmd)}</code>
+                    </div>
+                </div>`;
+
+            contentHTML += logCardTemplate
```

Comment:
can any of the replacements be there more than once every?

Reply:
my comment is bc .replace() only replaces the first occurence. I just wanted to double check if that is what you wanted (aka you only expect one instance)